packages/frontend-2/components/settings/workspaces/members/GuestsTable.vue

```diff
@@ -109,18 +117,26 @@
 </template>
 
 <script setup lang="ts">
-import {
+import type {
   WorkspaceSeatType,
-  type SettingsWorkspacesMembersActionsMenu_UserFragment
+  SettingsWorkspacesMembersActionsMenu_UserFragment
 } from '~/lib/common/generated/gql/graphql'
 import { Roles, type Nullable } from '@speckle/shared'
-import { settingsWorkspacesMembersSearchQuery } from '~~/lib/settings/graphql/queries'
+import {
+  settingsWorkspacesMembersSearchQuery,
+  settingsWorkspacesMembersTableQuery
+} from '~~/lib/settings/graphql/queries'
 import { usePaginatedQuery } from '~/lib/common/composables/graphql'
+import { useQuery } from '@vue/apollo-composable'
 
 const props = defineProps<{
   workspaceSlug: string
 }>()
 
+const { result } = useQuery(settingsWorkspacesMembersTableQuery, () => ({
```

Comment:
Split this up from the search query, so it doesnt fetch the whole workspace each time a search or inf load query is triggered